Sources/Overlays/_Testing_WinSDK/Attachments/_AttachableImageWrapper.swift

```diff
@@ -0,0 +1,132 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+#if os(Windows)
+@_spi(Experimental) public import Testing
+private import _TestingInternals.GDIPlus
+
+internal import WinSDK
+
+/// A wrapper type for image types such as `HBITMAP` and `HICON` that can be
+/// attached indirectly.
+///
+/// You do not need to use this type directly. Instead, initialize an instance
+/// of ``Attachment`` using an instance of an image type that conforms to
+/// ``AttachableAsGDIPlusImage``. The following system-provided image types
+/// conform to the ``AttachableAsGDIPlusImage`` protocol and can be attached to
+/// a test:
+///
+/// - [`HBITMAP`](https://learn.microsoft.com/en-us/windows/win32/gdi/bitmaps)
+/// - [`HICON`](https://learn.microsoft.com/en-us/windows/win32/menurc/icons)
+@_spi(Experimental)
+public struct _AttachableImageWrapper<Image>: ~Copyable where Image: AttachableAsGDIPlusImage {
+  /// The underlying image.
+  var image: Image
+
+  /// The image format to use when encoding the represented image.
+  var imageFormat: AttachableImageFormat?
+
+  /// Whether or not to call `_cleanUpAttachment(at:)` on `pointer` when this
+  /// instance is deinitialized.
+  ///
+  /// - Note: If cleanup is not performed, `pointer` is effectively being
+  ///   borrowed from the calling context.
+  var cleanUpWhenDone: Bool
```

Comment:
Should this perhaps be called `owning` (or `owned`)? Another alternative would be `consuming` (or `consumed`).

Reply:
🤷‍♂️ It's not public (actually it should be outright `private`) so I'm not too concerned there.